xbmc/FileItem.cpp

```diff
@@ -321,10 +321,11 @@ CFileItem::CFileItem(const CGenre& genre)
 }
 
 CFileItem::CFileItem(const CFileItem& item)
-: m_musicInfoTag(NULL),
-  m_videoInfoTag(NULL),
-  m_pictureInfoTag(NULL),
-  m_gameInfoTag(NULL)
+  : CGUIListItem(item),
+    m_musicInfoTag(NULL),
+    m_videoInfoTag(NULL),
+    m_pictureInfoTag(NULL),
+    m_gameInfoTag(NULL)
```

Comment:
Worthwhile changing these to nullptr while it's being moved?

Reply:
I think I'll change these to use `std::unique_ptr` in a future PR.